db/migrate/20161003145646_add_position_to_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddPositionToLinks < ActiveRecord::Migration
+  def change
+    add_column :links, :position, :integer, null: false, default: 0
```

Comment:
The reason this takes so long is the default setting. It'd be better to do the following:

1. Add the column, allowing nulls with no default
2. `UPDATE "links" SET "position" = 0 WHERE "position" IS NULL;`
3. Alter the column and add the default

That way, PostgreSQL won't block at all.